### PR TITLE
Subtitle for Mark Shuttleworth's speech at Fudan

### PR DESCRIPTION
**Pull request by yechs**

### CODE REVIEW

File: mark_shuttleworth_Xvid.ass [hunk 1]
```diff
@@ -0,0 +1,628 @@
...
+Dialogue: 0,0:01:43.07,0:01:45.37,Chinese,,0,0,0,,又有多少人想自己创业？
+Dialogue: 0,0:01:43.07,0:01:45.37,English,,0,0,0,,and how many of you want to start your own businesses
+Dialogue: 0,0:01:47.79,0:01:50.46,Chinese,,0,0,0,,我可以告诉你，在这群人中，谁会成为真正的富人
+Dialogue: 0,0:01:47.79,0:01:50.46,English,,0,0,0,,I can tell you who's gonna be really wealthy out of the crowd
+Dialogue: 0,0:01:50.46,0:01:53.79,Chinese,,0,0,0,,就是那些举手说要自己创业的人。
+Dialogue: 0,0:01:50.46,0:01:53.79,English,,0,0,0,,the people who put up their hands up to say they're gonna start their own businesses
+Dialogue: 0,0:01:56.02,0:02:01.51,Chinese,,0,0,0,,我今天想简单谈谈创业，因为
+Dialogue: 0,0:01:56.02,0:02:01.51,English,,0,0,0,,What I thought I would talk about today is entrepreneurship because
+Dialogue: 0,0:02:01.58,0:02:06.20,Chinese,,0,0,0,,我相信经济变革与技术变革同样重要
+Dialogue: 0,0:02:01.58,0:02:06.20,English,,0,0,0,,I believe that economical change is just as important as technological change
```
Comment (arsdragonfly) [inline mark_shuttleworth_Xvid.ass]: economical -> economic. Economical means "good value for the same amount of money".

File: mark_shuttleworth_Xvid.ass [hunk 2]
```diff
@@ -0,0 +1,628 @@
...
+Dialogue: 0,0:02:06.20,0:02:09.14,Chinese,,0,0,0,,而技术变革与社会变革同样重要。
+Dialogue: 0,0:02:06.20,0:02:09.14,English,,0,0,0,,which is just as important as social change
+Dialogue: 0,0:02:09.16,0:02:13.53,Chinese,,0,0,0,,而真正推动经济变革的是创业者
+Dialogue: 0,0:02:09.16,0:02:13.53,English,,0,0,0,,And it's entrepreneurs who really drive economical change
```
Comment (arsdragonfly) [inline mark_shuttleworth_Xvid.ass]: economical -> economic.

File: mark_shuttleworth_Xvid.ass [hunk 3]
```diff
@@ -0,0 +1,628 @@
...
+Dialogue: 0,0:02:55.51,0:03:01.75,Chinese,,0,0,0,,他们在过去10年里发生了根本性的变化。他们已经从根本上改变了人们对苹果的看法
+Dialogue: 0,0:02:55.51,0:03:01.75,English,,0,0,0,,They've radically changed in the last 10 years. They've radically changed the way people think about Apple
+Dialogue: 0,0:03:01.75,0:03:06.33,Chinese,,0,0,0,,10年前，也许每个人都认为苹果已经完了，而今天他们成为了所有人的宠儿
+Dialogue: 0,0:03:01.75,0:03:06.33,English,,0,0,0,,10 years ago maybe everybody thought Apple was dying and today they're the darling
+Dialogue: 0,0:03:06.33,0:03:11.47,Chinese,,0,0,0,,他们正在引领潮流。这是一个非常有趣的例子
+Dialogue: 0,0:03:06.33,0:03:11.47,English,,0,0,0,,They're setting the trend. It was a very interesting example
+Dialogue: 0,0:03:11.48,0:03:13.42,Chinese,,0,0,0,,还有哪些你欣赏的公司？
+Dialogue: 0,0:03:11.48,0:03:13.42,English,,0,0,0,,What are the other companies you admire?
+Dialogue: 0,0:03:13.42,0:03:20.40,Chinese,,0,0,0,,(听不清)另一家伟大的公司。但它不是一个新的公司，它没有真正改变事情
+Dialogue: 0,0:03:13.42,0:03:20.40,English,,0,0,0,,(inaudible) Another great company. But it's not a new company it's not really changing things
```
Comment (arsdragonfly) [inline mark_shuttleworth_Xvid.ass]: Might be "Intel".

File: mark_shuttleworth_Xvid.ass [hunk 4]
```diff
@@ -0,0 +1,628 @@
...
+Dialogue: 0,0:08:39.61,0:08:42.27,Chinese,,0,0,0,,基本上，他们的意思是，我们做的事情和其他人一模一样。
+Dialogue: 0,0:08:39.61,0:08:42.27,English,,0,0,0,,Basically, what they're saying is, we do exactly the same thing as everybody else.
+Dialogue: 0,0:08:42.27,0:08:45.05,Chinese,,0,0,0,,我们不做任何不同的事情，但我们更努力，对吗？
+Dialogue: 0,0:08:42.27,0:08:45.05,English,,0,0,0,,We don't do anything differently, but we try harder, right?
+Dialogue: 0,0:08:45.90,0:08:50.14,Chinese,,0,0,0,,现在我遇到的大多数企业家，他们都不属于这个类别。
+Dialogue: 0,0:08:45.90,0:08:50.14,English,,0,0,0,,Now most of the entrepreneurs that I meet, they don't fall into that category.
+Dialogue: 0,0:08:50.14,0:08:54.86,Chinese,,0,0,0,,他们不一定是最高效的管理者，或者说不一定
+Dialogue: 0,0:08:50.14,0:08:54.86,English,,0,0,0,,They're not necessarily extremely efficient managers, they're not necessarily extremely.
+Dialogue: 0,0:08:54.86,0:08:59.67,Chinese,,0,0,0,,具有最强的业务能力，但他们所做的是，他们创新。他们改变事情。
+Dialogue: 0,0:08:54.86,0:08:59.67,English,,0,0,0,,Operationly strong, but what they do is they innovate. They change things.
```
Comment (arsdragonfly) [inline mark_shuttleworth_Xvid.ass]: Operationally strong

File: mark_shuttleworth_Xvid.ass [hunk 5]
```diff
@@ -0,0 +1,628 @@
...
+Dialogue: 0,0:09:01.48,0:09:06.19,Chinese,,0,0,0,,我认为我这么多年来学到的可能是最大的教训，就是
+Dialogue: 0,0:09:01.48,0:09:06.19,English,,0,0,0,,Possibly the greatest lesson, I think that I've learned over the years is
+Dialogue: 0,0:09:06.19,0:09:11.09,Chinese,,0,0,0,,不管你做什么，大多数正在发生的变化都发生在社会中。
+Dialogue: 0,0:09:06.19,0:09:11.09,English,,0,0,0,,that most of the changes that are happening are changes that are happening in society, despite what you do.
+Dialogue: 0,0:09:11.09,0:09:12.61,Chinese,,0,0,0,,因为它是一个如此巨大的全球经济。
+Dialogue: 0,0:09:11.09,0:09:12.61,English,,0,0,0,,Because it's such a huge global economy.
+Dialogue: 0,0:09:14.10,0:09:16.53,Chinese,,0,0,0,,当然你仍然可以做出改变，可以创新
+Dialogue: 0,0:09:14.10,0:09:16.53,English,,0,0,0,,usually what's happening is that there is an underlying change going on in society.
```
Comment (arsdragonfly) [inline mark_shuttleworth_Xvid.ass]: While you can change things and you can innovate

File: mark_shuttleworth_Xvid.ass [hunk 6]
```diff
@@ -0,0 +1,628 @@
...
+Dialogue: 0,0:09:16.53,0:09:20.42,Chinese,,0,0,0,,但通常的情况是，社会正在潜移默化地改变。
+Dialogue: 0,0:09:16.53,0:09:20.42,English,,0,0,0,,usually what's happening is that there is an underlying change going on in society.
+Dialogue: 0,0:09:21.20,0:09:26.00,Chinese,,0,0,0,,因此，如果你了解这种潜在的变化，你清楚未来会如何变化
+Dialogue: 0,0:09:21.20,0:09:26.00,English,,0,0,0,,So if you understand that underlying change and you understand how the position is so forward.
```
Comment (arsdragonfly) [inline mark_shuttleworth_Xvid.ass]: So if you understand that underlying change and you understand how to position yourself for it.
......你清楚如何应对这种变化。

File: mark_shuttleworth_Xvid.ass [hunk 7]
```diff
@@ -0,0 +1,628 @@
...
+Dialogue: 0,0:11:15.54,0:11:20.19,Chinese,,0,0,0,,以同样的方式定价。如果你能以一个价格在英国生产一样商品，而在印度制造一些东西的价格只需要它的一半
+Dialogue: 0,0:11:15.54,0:11:20.19,English,,0,0,0,,ultimately be priced the same way that if you could manufacture something in the UK for one price
+Dialogue: 0,0:11:20.19,0:11:25.58,Chinese,,0,0,0,,那么所有生产它的产能都会转移（去到印度）
+Dialogue: 0,0:11:20.19,0:11:25.58,English,,0,0,0,,to manufacture something in India for half that price, that all of that manufacturing capacity move.
+Dialogue: 0,0:11:25.58,0:11:27.86,Chinese,,0,0,0,,因此，这是社会的一个巨大变化。
+Dialogue: 0,0:11:25.58,0:11:27.86,English,,0,0,0,,So that's one huge change that went through society.
+Dialogue: 0,0:11:28.93,0:11:33.45,Chinese,,0,0,0,,在我的一生中，社会的第二个巨大变化是互联网。
+Dialogue: 0,0:11:28.93,0:11:33.45,English,,0,0,0,,The second huge change that surfed through society in my lifetime was the Internet.
+Dialogue: 0,0:11:34.27,0:11:40.38,Chinese,,0,0,0,,这意味着信息可以在世界任何地方免费即时移动
+Dialogue: 0,0:11:34.27,0:11:40.38,English,,0,0,0,,It's effectively meant that information could move at no cost instantly anywhere in the world
```
Comment (arsdragonfly) [inline mark_shuttleworth_Xvid.ass]: It's -> which

File: mark_shuttleworth_Xvid.ass [hunk 8]
```diff
@@ -0,0 +1,628 @@
...
+Dialogue: 0,0:11:40.38,0:11:43.23,Chinese,,0,0,0,,起初，能够自由移动的是资本。
+Dialogue: 0,0:11:40.38,0:11:43.23,English,,0,0,0,,and at first it was capital that was able to move.
+Dialogue: 0,0:11:43.99,0:11:47.52,Chinese,,0,0,0,,而贯穿社会的下一个重大变化是信息
+Dialogue: 0,0:11:43.99,0:11:47.52,English,,0,0,0,,The next big change that scrape through society was information
```
Comment (arsdragonfly) [inline mark_shuttleworth_Xvid.ass]: scrape -> swept

File: mark_shuttleworth_Xvid.ass [hunk 9]
```diff
@@ -0,0 +1,628 @@
...
+Dialogue: 0,0:11:47.52,0:11:49.63,Chinese,,0,0,0,,使得信息也能够高效地移动，非常非常高效。
+Dialogue: 0,0:11:47.52,0:11:49.63,English,,0,0,0,,making information effectively move very, very efficient.
```
Comment (arsdragonfly) [inline mark_shuttleworth_Xvid.ass]: efficient -> efficiently

File: mark_shuttleworth_Xvid.ass [hunk 10]
```diff
@@ -0,0 +1,628 @@
...
+Dialogue: 0,0:16:17.84,0:16:21.63,Chinese,,0,0,0,,今天，音乐产业看起来仍然是中心化的。
+Dialogue: 0,0:16:17.84,0:16:21.63,English,,0,0,0,,Today, the music industry still looks like that centralized kind of thing.
+Dialogue: 0,0:16:21.63,0:16:27.05,Chinese,,0,0,0,,内容在中心生产，并向我所提到的边缘输送。
+Dialogue: 0,0:16:21.63,0:16:27.05,English,,0,0,0,,Content is produced at the center and then shipped out to the edges that I drew, right?
+Dialogue: 0,0:16:27.05,0:16:30.97,Chinese,,0,0,0,,而十年后或十五年后将会完全是相反的
+Dialogue: 0,0:16:27.05,0:16:30.97,English,,0,0,0,,So what's going to be what's going to be like in ten years time or fifteen year's time is the opposite
+Dialogue: 0,0:16:30.97,0:16:35.16,Chinese,,0,0,0,,内容将在网络的边缘生产，然后输送回（中心）。
+Dialogue: 0,0:16:30.97,0:16:35.16,English,,0,0,0,,content to be produced at the edges of the network and then shipped in.
+Dialogue: 0,0:16:35.16,0:16:39.38,Chinese,,0,0,0,,你将通过与你的朋友交谈来定义什么是流行音乐
+Dialogue: 0,0:16:35.16,0:16:39.38,English,,0,0,0,,out of that global conversation will come You will define what the popular music is by talking to your friends
```
Comment (arsdragonfly) [inline mark_shuttleworth_Xvid.ass]: ~~out of that global conversation will come~~

File: mark_shuttleworth_Xvid.ass [hunk 11]
```diff
@@ -0,0 +1,628 @@
...
+Dialogue: 0,0:21:53.13,0:22:01.20,Chinese,,0,0,0,,摩托罗拉说，它们 60% 的手机，在三年内都将会在 Linux 上运行。
+Dialogue: 0,0:21:53.13,0:22:01.20,English,,0,0,0,,And Motorola says that 60% of their phones, within three years, are all going to run on Linux.
+Dialogue: 0,0:22:01.20,0:22:06.30,Chinese,,0,0,0,,我们还不知道人们会用这些计算平台做什么。
+Dialogue: 0,0:22:01.20,0:22:06.30,English,,0,0,0,,We don't yet know what people will do with those computing platforms.
+Dialogue: 0,0:22:06.30,0:22:11.38,Chinese,,0,0,0,,所以我认为，多数创新、经济活动、以及下一代的大企业都将是
+Dialogue: 0,0:22:06.30,0:22:11.38,English,,0,0,0,,So I think that a lot of the innovation, a lot of the economic, the next generation of big businesses
+Dialogue: 0,0:22:11.38,0:22:17.31,Chinese,,0,0,0,,手持设备或由手持设备驱动
+Dialogue: 0,0:22:11.38,0:22:17.31,English,,0,0,0,,are effectively going to be handheld devices or powered by handheld device
+Dialogue: 0,0:22:17.31,0:22:19.36,Chinese,,0,0,0,,而不是由事件驱动
+Dialogue: 0,0:22:17.31,0:22:19.36,English,,0,0,0,,rather than being powered by incidents
```
Comment (arsdragonfly) [inline mark_shuttleworth_Xvid.ass]: incidents -> Internet

**Review by arsdragonfly — CHANGES_REQUESTED** [review]

File: mark_shuttleworth_Xvid.ass (hunk 3 above)
Comment (yechs) [reply mark_shuttleworth_Xvid.ass]: Hmm. Could be

File: mark_shuttleworth_Xvid.ass (hunk 6 above)
Comment (yechs) [reply mark_shuttleworth_Xvid.ass]: Cool! 这个地方真的没听明白

**Pull request merged**